packages/neutron-sdk/src/stargate/aux.rs

```diff
@@ -0,0 +1,43 @@
+use cosmwasm_std::{Binary, CosmosMsg, Deps, QueryRequest, StdResult};
+use prost_types::Timestamp as TimestampGen;
+use serde::de::DeserializeOwned;
+
+/// makes a stargate query by a given path and req and returns a response deserialised into a
+/// given response model.
+///
+/// * **req** is a proto request model. Most likely it's a result of proto code generation;
+/// * **path** is an RPC request path. Should be one of allowlisted stargate query paths;
+///
+/// Since stargate query results are JSON-encoded instead of protobuf-encoded, the Res is
+/// expected to have a serde::de::DeserializeOwned trait. Why JSON, not proto? See the link:
+/// https://github.com/CosmWasm/wasmd/blob/6f6be7880f1caa666b86aaafea625208d70675dc/x/wasm/keeper/query_plugins.go#L360
+pub fn make_stargate_query<Req, Res>(deps: Deps, req: Req, path: &str) -> StdResult<Res>
```

Comment:
I think it's more intuitive to have path in the first place and message in the second. A bit misleading